Makefile

```diff
@@ -2,9 +2,6 @@ all: prelaunch-3 regen-1
 
```

Comment:
We should also drop `prelaunch-3` from line 1, and line 3.

Reply:
My bad :facepalm: , updated now.